contracts/Colony.sol

```diff
@@ -23,9 +23,17 @@ import "./ERC20Extended.sol";
 import "./IColonyNetwork.sol";
 import "./IColony.sol";
 import "./ColonyStorage.sol";
+import "./PatriciaTree/Data.sol";
+import {Bits} from "./PatriciaTree/Bits.sol";
 
 
 contract Colony is ColonyStorage {
+  using Bits for uint;
```

Comment:
If we ever need to support 2 versions of these libraries it'll be a problem as they are naturally singletons. The only way I can think of is to rename new versions, e.g. `Bits2`, `Data2` etc

Reply:
By 'support', do you mean on-chain, or only on the repository level? I'm struggling to come up with a scenario where a contract would need to know about both of these hypothetical versions.

Reply:
I was thinking about different `Colony` contract versions referencing different library versions. I guess with library linking this is ok, maybe I'd just forgotten about that here.